python/tvm/relay/graph_runtime/__init__.py

```diff
@@ -0,0 +1,2 @@
+from . import codegen
```

Comment:
we do not need the graph_runtime subfolder for now, this could also be confusing, as it is more of a compiler rather than "runtime", tvm's graph_runtime is a runtime. Let us consider use the old name graph_runtime_codegen for now